platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -91,10 +91,10 @@ const _isQueryParamApplied = (study, filters = {}, isFilterStrategy) => {
     return applied;
   }
 
-  const { seriesList = [], displaySets = [] } = study;
-  const firstSeries = isFilterStrategy ? seriesList[0] : displaySets[0];
+  const { series = [], displaySets = [] } = study;
```

Comment:
Do we need to worry about accidental conflicts here?

Reply:
I've tested the app thoroughly after these changes and haven't encounted any yet.